tools/DfmHttpService/Model/CommandMessage.cs

```diff
@@ -19,16 +19,19 @@ internal class CommandMessage
         [JsonProperty("markdownContent")]
         public string MarkdownContent { get; set; }
 
-        [JsonProperty("writeTempPreviewFile")]
-        public bool WriteTempPreviewFile { get; set; }
+        [JsonProperty("shouldWriteTempPreviewFile")]
+        public bool ShouldWriteTempPreviewFile { get; set; }
 
-        [JsonProperty("previewFilePath")]
-        public string PreviewFilePath { get; set; }
+        [JsonProperty("shouldSeparateMarkupResult")]
+        public bool ShouldSeparateMarkupResult { get; set; }
+
+        [JsonProperty("tempPreviewFilePath")]
+        public string TempPreviewFilePath { get; set; }
 
         [JsonProperty("pageRefreshJsFilePath")]
         public string PageRefreshJsFilePath { get; set; }
```

Comment:
remove this three parameter, read original html and write temp html file in client.